services/transition/Makefile

```diff
@@ -0,0 +1 @@
+transition: bundle-transition
```

Comment:
Can we setup the database here?

https://github.com/alphagov/govuk-docker/blob/master/services/content-publisher/Makefile#L2